config/locales/doc_auth/en.yml

```diff
@@ -201,8 +201,10 @@ en:
         considered valid. Do not enter real PII in this field.
       text1: Your ID cannot be expired.
       text2: You will not need the card with you.
-      text3: We’ll call or text this number to verify your identity. If we can’t
-        verify your phone number, you can verify by mail instead.
+      text3_html: '<strong>Verify by phone</strong>: We’ll call or text your phone
+        number. This takes a few minutes.'
+      text4_html: '<strong>Verify by mail</strong>: We’ll mail a letter to your home
```

Comment:
I think the numbering here was meant to correspond to the `bullet3` heading, so I might expect something like `text3a` and `text3b` instead of `text3` and `text4`.

Elsewhere, we've also stopped enumerating strings like this and using arrays instead, since it avoids some of the common headaches with numbering manually. Not sure if it'd be a great fit here though.

Example:

https://github.com/18F/identity-idp/blob/d387ce2ff710b4f30fb01dc4357f9134bb6dcaef/app/views/idv/in_person/ready_to_verify/show.html.erb#L78-L80

Reply:
Yes, good idea.